js/models/localattachment.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ * @author Luc Calaresu <dev@calaresu.com>
+ *
+ * ownCloud - Mail
```

Comment:
Oups hahaha, thanks for that! I double checked the headers, but it seems I saw what I wanted to see :stuck_out_tongue: